concepts/05 UI Components/Scheduler/040 Resources/040 Customize Resource Headers.md

```diff
@@ -48,6 +46,8 @@ For AngularJS and Knockout apps, DevExtreme provides the [dxTemplate](/api-refer
 
 ##### Vue
 
+Templates allow you to customize UI components. DevExtreme templates make use of the functionality provided by Vue. Implement a custom template and assign it to the [resourceCellTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#resourceCellTemplate) property as shown in the following code.
```

Comment:
There is something strange about your second sentence here. Can you rewrite it so that there is a better flow of information and a clearer relationship between the first sentence and the second. You can probably just combine these into one sentence.

Reply:
DevExtreme templates allow you to customize UI components and make use of the functionality provided by Vue. Implement a custom template and assign ...